src/Server/SSH/SSHBind.cpp

```diff
@@ -0,0 +1,78 @@
+#include "SSHBind.h"
+#include <stdexcept>
+#include <fmt/format.h>
+#include <Common/SSH/clibssh.h>
+
+namespace ssh
+{
+
+SSHBind::SSHBind() : bind_(ssh_bind_new(), &deleter)
+{
+    if (!bind_)
+    {
+        throw std::runtime_error("Failed to create ssh_bind");
```

Comment:
use `DB::Exception`

Reply:
Changed in `Make ssh objects throw DB::Exception` commit